apps/election-manager/src/lib/__snapshots__/votecounting.test.ts.snap

```diff
@@ -24041,139 +24041,139 @@ Array [
 exports[`filterTalliesByParams in a typical election can filtere by precinct and scanner 2`] = `
 Object {
   "Chief-Justice-1": Object {
-    "ballots": 0,
-    "overvotes": 0,
-    "undervotes": 0,
+    "ballots": 144,
```

Comment:
Unfortunately this was a bug from my last PR that my slight refactor here just happened to also fix. (I'm going to do some more involved testing of all of these tabulation changes tomorrow after discovering this).